system/Validation/Rules.php

```diff
@@ -364,10 +364,10 @@ public function required_without($str = null, string $fields, array $data): bool
 		}
 
 		// Still here? Then we fail this test if
-		// any of the fields are not present in $data
+		// any of the fields are not present or value is '' in $data
 		foreach ($fields as $field)
 		{
-			if (! array_key_exists($field, $data))
+			if (! array_key_exists($field, $data) || ! $this->required($data[$field] ?? ''))
```

Comment:
This change is not required as we just intend to check here whether the field exists in the data or not. It doesn't matter whether the field is required or not.